docs/docs-ref-autogen/excel/excelscript/excelscript.filterpivothierarchy.yml

```
@@ -60,7 +86,25 @@ methods:
       content: 'getName(): string;'
       return:
         type: string
-        description: ''
+        description: |-
+
+
+          #### Examples
+
+          ```TypeScript
+          /**
+           * This script logs the names of all the filter hierarchies a PivotTable.
+           */
+          function main(workbook: ExcelScript.Workbook) {
+            // Get the first PivotTable in the workbook.
```

It looks like the comment isn't aligned with the following line of code, so adding a space
```suggestion
             // Get the first PivotTable in the workbook.
```